### PR TITLE
Restore missing quotations from Dr. Sant Singh Khalsa English Translation

### PR DESCRIPTION
**Pull request by Sarabveer**
Restore missing quotations from Dr. Sant Singh Khalsa's English Translation. Many quotations (indicating conversations or speech) were missing, most possibly an error during importing of the database.

Also removed capitalization from sentences that span multiple lines to stay close to the original translation.

These were supposed to be 2 separate commits.

Reference: English Translation of SGGS - One Volume (Dr. Sant Singh Khalsa).pdf

### CODE REVIEW

File: data/Sri Guru Granth Sahib Ji/0092.json [hunk 1]
```diff
@@ -339,7 +339,7 @@
         "translations": {
           "English": {
             "Dr. Sant Singh Khalsa": {
-              "translation": "O my Lord, please save me!||1||Pause||",
+              "translation": "O my Lord, please save me!\" ||1||Pause||",
```
Comment (Harjot1Singh) [inline data/Sri Guru Granth Sahib Ji/0092.json]: It’s a bit weird that quotes can span multiple lines

Comment (Sarabveer) [reply data/Sri Guru Granth Sahib Ji/0092.json]: Yep

**Comment by HarminderSingh** [comment]
Hi, I was looking for a correct copy of the Gurbani. I notice your work and make account. I very surprised at all work you doing, but really amazing that you confirm 3,000 lines in only 2 hrs

**Comment by bhajneet** [comment]
> Hi, I was looking for a correct copy of the Gurbani. I notice your work and make account. I very surprised at all work you doing, but really amazing that you confirm 3,000 lines in only 2 hrs

Your words are encouraging. If you'd like to learn more or discuss please message us using the contact info on our [GitHub profile](https://github.com/ShabadOS/).

**Comment by Sarabveer** [comment]
This was done by having a script go through a text dump of the Shabad OS Dr. Sant Singh Khalsa translation and a text dump of the PDF.

**Comment by HarminderSingh** [comment]
@bhajneet how did you approve so many so fast - i want to learn how to do the same only. is any other team doing same work?

**Pull request merged**